enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.unit.spec.ts

```diff
@@ -24,7 +24,9 @@ describe("Transform Embedding Theme Override", () => {
       colors: {
         brand: expect.arrayContaining(["hotpink"]),
         "text-dark": expect.arrayContaining(["yellow"]),
+        "text-primary": expect.arrayContaining(["yellow"]),
```

Comment:
You can see from the new mapping that `text-primary` will map to both:
- `text-dark`
- `text-primary`

https://github.com/metabase/metabase/blob/7b68b42d9452afd4db37fc6d776d556d38113fde/enterprise/frontend/src/embedding-sdk/lib/theme/embedding-color-palette.ts#L41